docs/index.md

```diff
@@ -1,3 +1,34 @@
-Welcome to Lockbox's extension documentation!
+<style>
+div.install-content {
+  text-align: center;
+}
 
-*(INSERT PICTURE AND TOC HERE?)*
+div.install-content * {
+  font-family: sans-serif;
+}
+
+div.install-content p {
+  margin-bottom: 2em;
+  font-weight: bold;
+}
+
+div.install-content > a {
+  display: block;
+  margin: 0px auto;
+  padding: 0.5em;
+  width: 12em;
+  color: white;
+  background-color: #717171;
+  border: 1px solid #717171;
+  border-radius: 4px;
+}
+</style>
+
+<div class='install-content'>
+  <h1>Lockbox Extension</h1>
+
+  <p>Click below to install the Lockbox extension:</p>
+
+  <a href='https://testpilot.firefox.com/files/lockbox@mozilla.com/latest'>install</a>
```

Comment:
Should this point to the latest signed TestPilot add-on, or since this is for the UR team should we host the XPI somewhere more stable so it doesn't get potentially auto updated so it isn't tied to the alpha release?

Reply:
From talking with the UR team, they think this should be fine.